packages/waku/src/lib/plugins/vite-plugin-rsc-hmr.ts

```diff
@@ -122,6 +123,21 @@ export function rscHmrPlugin(): Plugin {
         );
       }
     },
+    handleHotUpdate({ file }) {
+      const moduleLoading = (globalThis as any).__webpack_module_loading__;
+      const moduleCache = (globalThis as any).__webpack_module_cache__;
+      const id = filePathToFileURL(file);
+      if (moduleLoading.has(id) && moduleCache.has(id)) {
```

Comment:
Should we check only the first one?
```suggestion
      if (moduleLoading.has(id)) {
```

Reply:
I do not know about react's impl details, but I think checking the two might be safer?

Reply:
It's not React impl, but yeah it can be a little safer. Let's leave it.